Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -0,0 +1,155 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgeDescriptionCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var containerImageView: UIView = { UIView.init() }()
+  private lazy var pledgeImage: UIImageView = { UIImageView(frame: .zero) }()
+  private lazy var descriptionStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var estimatedDeliveryLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var dateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var descriptionLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var learnMoreLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var spacerView: UIView = { UIView(frame: .zero) }()
+
+  func configureWith(value: String) {
```

Comment:
Can we follow the other work and place this bellow `Lifecycle` under its own `Configuration` pragma mark section?